README.md

```diff
@@ -128,16 +128,15 @@ The following properties are accepted:
 - `nodeVersion`
 
   - _Type_: `string`\
-  - _Default value_: `16.x`
+  - _Default value_: `16`
 
-  The version of Node.js to use as the compilation target. Possible values:
+  The version of Node.js to use as the compilation target and reported execution runtime. Any valid and supported
```

Comment:
Curious what "reported" means in this context?

Reply:
changed